src/components/game/Board.jsx

```diff
@@ -1,40 +1,20 @@
-import React, { useState } from "react";
+import React from "react";
```

Comment:
You can even remove `import React from "react";` here because this is no longer needed in functional components after React v17.